### ISSUE TITLE
Segmentation fault when i run the run.sh

### ISSUE THREAD

**Issue by dingsiyu** [donglixp/coarse2fine#18]
**os information**:   the same with yours
cuda : 8.0
cudnn : 5.1

** issue**:
run.sh: line 27: 17688 Segmentation fault      (core dumped) CUDA_VISIBLE_DEVICES=$GPU_ID python train.py -root_dir "$WORK_DIR/data_model/" -dataset $DATANAME -rnn_size 250 -word_vec_size 200 -decoder_input_size 150 -layers 1 -start_checkpoint_at 60 -learning_rate 0.005 -start_decay_at 0 -epochs 100 -global_attention "dot" -attn_hidden 0 -lock_dropout -dropout 0.5 -dropout_i 0.5

can you tell me what happened ? and how can i address it ?

**Comment by ygan** [comment]
You can try comment or delete: from tensorboard_logger import Logger

**Comment by Don-Joey** [comment]
thx，but why？ @ygan
